readme/tutorial/Semantic.scalatex

```diff
@@ -9,89 +9,28 @@
   symbol signatures (@code{_root_.com.Example.main([Ljava/lang/String;)V.} =>
   @code{def main(args: Array[String]): Unit}),
   These operations can for example be used by tools like
-  @lnk("scalafix", "https://scalacenter.github.io/scalafix") to rewrite Scala source code.
+  @lnk("Scalafix", "https://scalacenter.github.io/scalafix"),
+  @lnk("Metadoc", "http://scalameta.org/metadoc/"),
+  @lnk("Metals", "https://geirsson.com/assets/metals/").
 
   @sect{SemanticDB}
     @p
-      A key property of the Scalameta semantic API is its @b{portability}.
-      "SemanticDB" is the storage format for information that
-      the semantic API consumes.
-
-    @p
-      Currently, a typical approach to semantic developer tools in Scala is
-      implementing them as compiler plugins and then using them inside builds.
-      Apart from being a hassle to configure, this approach is also quite slow,
-      because it needs to run a Scala typechecker every time when a tool is
-      invoked.
-
-    @p
-      Scalameta enables a much more convenient workflow. First, we use our
-      semanticdb-scalac compiler plugin to generate a semantic database for it
-      during type checking.
-      This is done only once per unique snapshot of the codebase.
-      Afterwards, using the persisted semantic data, we can launch
-      any number of developer tools any number of times without having to
-      typecheck the codebase again.
+      SemanticDB is a data model for semantic information about programs in Scala and
+      other languages. SemanticDB decouples production and consumption of semantic information,
+      establishing documented means for communication between tools.
 
     @p
       The storage format used for the SemanticDB is defined using
-      @lnk("Protocol Buffers", "https://www.google.ch/search?q=protocol+buffers&oq=protocol+buffer&aqs=chrome.0.0j69i61j69i57j69i61j0l2.6856j0j7&sourceid=chrome&ie=UTF-8"),
+      @lnk("Protocol Buffers", "https://developers.google.com/protocol-buffers/"),
       or "protobuf" for short.  The SemanticDB protobuf schema is small, at
-      the time of this writing it is ~50 lines of code.  The full schema is
+      the time of this writing it is ~60 lines of code.  The full schema is
       available
       @lnk("here",
-           "https://github.com/scalameta/scalameta/blob/master/langmeta/shared/src/main/protobuf/semanticdb.proto").
+           "https://github.com/scalameta/scalameta/blob/master/semanticdb/semanticdb2/semanticdb2.proto").
```

Comment:
We should really try to stop moving this file around, I think this is like the 5th location we keep it at.

Reply:
😅